kitchen.recipes.yml

```diff
@@ -15,6 +15,12 @@ verifier:
     - test/recipes
     - test/resources
 
+lifecycle:
```

Comment:
Question (non-blocking): is the lifecycle executable in parallel?

Reply:
Do you mean in parallel across suites? Yes, it is.